src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -590,10 +685,9 @@ internal async void ExecutePackageDownload(string name, PackageVersion package,
                     }
                 }).ToList();
 
-                // if any header download fails, abort
-                if (dependencyVersionHeaders.Any(x => x == null))
```

Comment:
Don't we still have to check for this condition?

Reply:
we do not since we now add the main package if it does not exist
```
 if (!dependencyVersionHeaders.Any(x => x.name == name))
{// Add the main package if it does not exist
   dependencyVersionHeaders.Add(package);
 ```